### DISCUSSION TITLE
Hide or change redundant data

### DISCUSSION THREAD

**Discussion by JPT62089** [filamentphp/filament#12739] [Help]
### Package

Table builder

### Package Version

v3.2.73

### How can we help you?

Howdy! I can't seem to find any resources on this or my search fu sucks.

I'm developing a table of information where there is a lot of repeat information. Is there any way to hide or replace the data in a column on a row if the previous row has the same data in the same column? Maybe I'm overthinking it or something haha.

Basically this is what we have:

![image](https://github.com/filamentphp/filament/assets/1808107/14eecf3d-36eb-4012-b201-b7c4c1929612)

And this is what we're looking for:
![image](https://github.com/filamentphp/filament/assets/1808107/f4ede798-4087-4a02-8529-c87ca3b2f8c7)

or

![image](https://github.com/filamentphp/filament/assets/1808107/2d7f0610-d2c9-4fa5-ae5f-5625e76eda6c)
